qiskit/circuit/parametervector.py

```diff
@@ -12,9 +12,45 @@
 
 """Parameter Vector Class to simplify management of parameter lists."""
 
+from uuid import uuid4
+
 from .parameter import Parameter
 
 
+class ParameterVectorElement(Parameter):
+    """An element of a ParameterVector."""
+
+    def __new__(cls, vector, index, uuid=None):  # pylint:disable=unused-argument
```

Comment:
This looks good. Can you add a test (similar to https://github.com/Qiskit/qiskit-terra/blob/d396a91520eb749a3a82fd6e04847690caa38a9c/test/python/circuit/test_parameters.py#L591 , or include as a case there) to check that `ParameterVectorElement.__eq__` still works across serialization and de-serialization.

Reply:
Added in 176b2069d89aa26dc1b81734c1fe31377f87dfb0